helusers/oidc.py

```diff
@@ -30,68 +31,70 @@ def keys(self):
         return requests.get(keys_url).json()
 
 
-def _build_defaults():
-    class _Defaults:
-        @cached_property
-        def audience(self):
-            if not api_token_auth_settings.AUDIENCE:
-                raise ImproperlyConfigured(
-                    "You must set OIDC_API_TOKEN_AUTH['AUDIENCE'] setting to the accepted JWT audience."
-                )
-            return api_token_auth_settings.AUDIENCE
-
-        @cached_property
-        def issuers(self):
-            issuers = api_token_auth_settings.ISSUER
-            if not issuers:
-                raise ImproperlyConfigured(
-                    "You must set OIDC_API_TOKEN_AUTH['ISSUER'] setting to one or to a list of accepted JWT issuers."
-                )
-            if isinstance(issuers, str):
-                issuers = [issuers]
-            return issuers
+class Config:
```

Comment:
After making this class public and calling it `Config`, this `oidc` module now has both `OIDCConfig` and `Config` classes. That could be somewhat confusing for the user.

Reply:
I reverted the decision to make this public `Config` class and kept it as it was before. Instead added two new functions (`get_keys` and `accepted_audience`) into the `oidc` module that can be used to fetch some OIDC related data.